meta.yml

```
@@ -0,0 +1,138 @@
+---
+fullname: Topology
+shortname: topology
+organization: coq-community
+
+synopsis: General Topology
+
+description: |
+  This library develops some of the basic concepts and results of general topology.
+
+authors:
+- name: Daniel Schepler <dschepler@gmail.com>
+  initial: true
+
+maintainers:
+- name: Andrew Miloradovsky
+  nickname: amiloradovsky
+
+opam-file-maintainer: miloradovsky@gmail.com
+
+license:
+  fullname: GNU Lesser General Public License v2.1 or later
+  identifier: LGPL-2.1-or-later
+  file: COPYING
+
+supported_coq_versions:
+  text: Coq 8.6 or later (use the corresponding branch or release for other Coq versions)
+  opam: '{(>= "8.6" & < "8.10~") | (= "dev")}'
+
+tested_coq_versions:
+- version_or_url: https://github.com/coq/coq/tarball/master
+- version_or_url: 8.9
+- version_or_url: 8.8
+- version_or_url: 8.7
+- version_or_url: 8.6
+
+tested_coq_opam_version: dev
+
+dependencies:
+- opam:
+    name: coq-zorns-lemma
+    version: '{(>= "8.6" & < "8.10~") | (= "dev")}'
+  nix: coq-zorns-lemma
+  description: |
+    [Zorn's lemma](https://github.com/coq-community/zorns-lemma)
+
+namespace: Topology
+
+keywords:
+- name: topology
+- name: filters
+- name: nets
+- name: metric spaces
+- name: real analysis
+- name: Urysohn's lemma
+- name: Tietze extension theorem
+
+categories:
+- name: Mathematics/Real Calculus and Topology
+
+documentation: |
+  Contents, roughly grouped in related categories:
+
+  # Basic definitions
```

```suggestion
  ### Basic definitions
```

and same for the following sub-sections.